common/src/lib.rs

```diff
@@ -158,6 +158,9 @@ pub enum CrucibleError {
 
     #[error("missing block context for non-empty block")]
     MissingBlockContext,
+
+    #[error("Incompatable RegionDefinition {0}")]
```

Comment:
`s/Incompatable/Incompatible/g`

Reply:
Done in [7bef555](https://github.com/oxidecomputer/crucible/commit/7bef55560bd1a345855ca77b6c296616ee66e0fe)